xapi_db_load/generate_load.py

```diff
@@ -90,12 +90,14 @@ class RandomCourse:
     start_date = None
     end_date = None
 
-    def __init__(self, org):
+    def __init__(self, org, min_start_date, max_end_date):
```

Comment:
nit: why are these called `min_start_date` and `max_end_date` when everywhere else, they're called `start_date` and `end_date`?